tests/queries/0_stateless/02496_remove_redundant_sorting_analyzer.reference

```diff
@@ -220,10 +220,8 @@ GROUP BY number
 -- explain
 Expression ((Project names + Projection))
   Aggregating
-    Expression ((Before GROUP BY + (Change column names to column identifiers + Project names)))
```

Comment:
Why did it change?

Reply:
For the follow query match `SELECT any(column) FROM table GROUP BY column` pattern and `any` is removed.
```
SELECT any(number)
FROM
(
    SELECT *
    FROM
    (
        SELECT *
        FROM numbers(3)
        ORDER BY number ASC
    )
    ORDER BY number DESC
)
GROUP BY number
```

And `any` is order dependent, so the result is different.